docs/faq/wallets-and-seeds.md

```diff
@@ -70,27 +70,7 @@ A public key address, also called Pay-To-Pubkey (P2Pk), can be identified with i
 
 #### 8. What are the different types of addresses?
```

Comment:
Seeing as we now have both the glossary entry and the whole page dedicated to addresses, do you think we could just drop this FAQ entry altogether? To me it feels like we have enough content on this subject and we dont need the FAQ as well

Reply:
If this was only in a page under /Advanced, I'd say keep it, as it's not easily discoverable. But with the address definition in the glossary (which links to this page), I think we should remove this (have removed in last push, but can add back if someone strongly disagrees).